language/changes/7-packages.md

```diff
@@ -0,0 +1,325 @@
+# Packages
+
+* Status: Implemented in Move 1.5
+
+## Introduction
+
+Packages are a new source feature to allow Move programmers to more easily
+re-use code and share it across projects.  Moreover, with a large Move
```

Comment:
Does the "moreover...." sentence still make sense here?

Reply:
It reads better without. Removed.